packages/amplify-graphql-index-transformer/package.json

```diff
@@ -27,18 +27,18 @@
     "test": "jest"
   },
   "dependencies": {
-    "@aws-amplify/graphql-model-transformer": "0.16.0",
-    "@aws-amplify/graphql-transformer-core": "0.17.12",
-    "@aws-amplify/graphql-transformer-interfaces": "1.14.7",
-    "@aws-cdk/aws-appsync": "~1.124.0",
-    "@aws-cdk/aws-dynamodb": "~1.124.0",
-    "@aws-cdk/core": "~1.124.0",
+    "@aws-amplify/graphql-model-transformer": "1.0.0",
+    "@aws-amplify/graphql-transformer-core": "1.0.0",
+    "@aws-amplify/graphql-transformer-interfaces": "2.0.0",
+    "aws-cdk-lib": "^2.41.0",
+    "@aws-cdk/aws-appsync-alpha": "^2.41.0-alpha.0",
+    "constructs": "^10.0.5",
     "graphql": "^14.5.8",
     "graphql-mapping-template": "4.20.5",
     "graphql-transformer-common": "4.24.0"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "~1.124.0"
+    "@aws-cdk/assert": "~2.41.0"
```

Comment:
Doesn't it comes under 'aws-cdk-lib' ?

Reply:
thereis this : https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.assertions-readme.html

Reply:
So this is indeed part of core package. I'll work on this then. thanks!